packages/pluggableWidgets/radio-buttons-native/src/RadioButtons.editorConfig.ts

```diff
@@ -0,0 +1,132 @@
+import { StructurePreviewProps, Properties, hidePropertiesIn } from "@mendix/piw-utils-internal";
+import { RadioButtonsPreviewProps } from "../typings/RadioButtonsProps";
+import darkRadioIcon from "./assets/radioButton_dark.svg";
+import lightRadioIcon from "./assets/radioButton_light.svg";
+import lightUnselectedRadioIcon from "./assets/radioButton_unselected_light.svg";
+import darkUnselectedRadioIcon from "./assets/radioButton_unselected_dark.svg";
+
+export function getPreview(values: RadioButtonsPreviewProps, isDark: boolean): StructurePreviewProps {
+    const radioButtonLabel: StructurePreviewProps = {
+        type: "Container",
+        borders: false,
+        children: [
+            {
+                type: "Text",
+                content: values.label !== "" ? values.label : "Label",
+                fontColor: isDark ? "#D6D6D6" : "#0A1324"
```

Comment:
These hexes are repeated it seems, I think variabels are suited

Reply:
Done